model/scikit/tests/test_scikit.py

```diff
@@ -178,6 +208,58 @@ def tearDownClass(cls):
     [21.20000076, 18.60000038, 8.7, 112481.0],
 ]
 
+FEATURE_DATA_CLUSTERING = [
+    [-9.01904123, 6.44409816, 5.95914173, 6.30718146],
```

Comment:
Just to be sure, this is randomly generated data right? If not let's make sure it is.

Reply:
I would suggest used a  seed and generate it on the fly so that the source is clear, or do inline doc

Reply:
@pdxjohnny this is randomly generated, anyways I pushed the changes to generate data on the fly by changing seeds.